### PR TITLE
test: Cypress - json form widget tests 

### PR DESCRIPTION
**Pull request by sharanya-appsmith**

### CODE REVIEW

File: app/client/cypress/support/Pages/PropertyPane.ts [hunk 1]
```diff
@@ -162,6 +162,7 @@ export class PropertyPane {
     ".currency-change-dropdown-trigger .remixicon-icon";
   _countryCodeChangeDropDown = ".t--input-country-code-change .remixicon-icon";
   _searchCountryPlaceHolder = "[placeholder='Search by ISD code or country']";
+  _closeModal = "//*[contains(@class,'ads-v2-button t--close')]";
```
Comment (NandanAnantharamu) [inline app/client/cypress/support/Pages/PropertyPane.ts:165]: there is one cloaseModal locator ,   _closeModal = "button:contains('Close')" in CommonLocators check if this works for your usecase

File: app/client/cypress/support/Pages/Table.ts [hunk 2]
```diff
@@ -198,6 +198,8 @@ export class Table {
   _paginationItem = (value: number) => `.rc-pagination-item-${value}`;
   _cellWrapOff = "//div[@class='tableWrap virtual']";
   _cellWrapOn = "//div[@class='tableWrap']";
+  _multirowselect = ".t--table-multiselect";
+  _selectedrow = ".selected-row";
```
Comment (NandanAnantharamu) [inline app/client/cypress/support/Pages/Table.ts:202]: _tableSelectedRow is present in this file, just see if that works fr your usecase

**Review by NandanAnantharamu — DISMISSED** [review]
LGTM

File: app/client/cypress/support/Pages/PropertyPane.ts (hunk 1 above)
Comment (sharanya-appsmith) [reply app/client/cypress/support/Pages/PropertyPane.ts:165]: this is to close a modal in property pane which is visible on selecting show alert or similar function. so button:contains('Close') isn't working.

**Review by NandanAnantharamu — APPROVED** [review]
Lgtm

**Pull request merged**